ansible/roles/runner/tasks/setup.yml

```diff
@@ -73,6 +73,22 @@
   when: vagrant_version.stdout is version('2.2.0', '<')
   ignore_errors: yes
 
+# TODO: Replace COPR repo when the packages hit the stable repos
+- name: install winrm packages and dependencies for AD tests
+  block:
+    - name: enable winrm copr repo
+      shell: "dnf copr enable -y abiagion/winrm"
```

Comment:
Instead of using a personal copr repo, @freeipa/freeipa-master could be leveraged.

Reply:
Nice, I've requested `builder` and `admin` permissions to that repo.

Reply:
Packages added to [@freeipa/freeipa-pr-ci](https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-pr-ci/packages/) repo.